### PR TITLE
Docs for Openstack Support and alternate default certificate

### PR DESCRIPTION
**Pull request by aaha97**

### CODE REVIEW

File: docs/public_clouds.md [hunk 1]
```diff
@@ -113,6 +113,30 @@ Following lists the Day 0 preparation work required to set up AKO in Azure
         * `save`
     * Similarly override the `se_subnet_id` for the `cluster2` .
 
+## AKO in OpenStack
+
+AKO can be deployed with AVI on an Openstack cloud.
```
Comment (chauhanshubham) [inline docs/public_clouds.md:118]: consider **in an** ?

File: docs/public_clouds.md [hunk 2]
```diff
@@ -113,6 +113,30 @@ Following lists the Day 0 preparation work required to set up AKO in Azure
...
+
+### AVI side configuration
+
+Configure Openstack cloud on AVI controller. [link](https://avinetworks.com/docs/21.1/installing-avi-vantage-for-openstack/)
+
+The Openstack tenants are synced with AVI tenants automatically.
+
+### AKO side configuration
+
+The vipNetworkList requires network id to be specified and not the name of the network. The values yaml can be updated as follows
```
[the remark this answers is not in the document]
Comment (aaha97) [reply docs/public_clouds.md]: updated

File: docs/ingress/ingress.md [hunk 3]
```diff
@@ -68,6 +68,7 @@ This feature can be used when the user wants to apply a common key-cert for mult
 
 To use this feature: 
 - A Secret with name `router-certs-default` has to be created in the same namespace where AKO pod is running (avi-system). The Secret must have tls.crt and tls.key fields in its data section.
+- Additonally, alt.crt and alt.key can be fields can be populated to allow multiple default certificates when trying to configure both RSA and ECC signed certificates. Avi Controller allows a Virtual Service to be configured with two certificates at a time, one each of RSA and ECC. This enables Avi Controller to negotiate the optimal algorithm or cipher with the client. If the client supports ECC, in that case the ECC algorithm is preferred, and RSA is used as a fallback in cases where the clients do not support ECC.
```
[the remark this answers is not in the document]
Comment (aaha97) [reply docs/ingress/ingress.md]: updated

**Comment by chauhanshubham** [comment]
build ako

**Review by akshayhavile — APPROVED** [review]

**Pull request merged**